README.md

```diff
@@ -393,6 +448,26 @@ rosrun zivid_samples sample_capture_2d_cpp
 rosrun zivid_samples sample_capture_2d.py
 ```
 
+### Sample Capture Assistant
```

Comment:
This sample doesn't continue capturing in a loop like `sample_capture`. Why not?

Reply:
As discussed IRL, the reason the other ones are in a loop is to make it easy to change settings and visualize the output. That is not so relevant for this sample. Also, having it in a loop IMO will make it more difficult to understand how the CA works with the initial captures and then the actual capture, fex. observing how long the two parts take.

Reply:
You make good points. I agree now. :)